cmake_format/doc/README.rst

```diff
@@ -26,8 +26,29 @@ Integrations
 * There is an official `vscode extension`__
 * Someone also created a `sublime plugin`__
 
+pre-commit tool
+~~~~~~~~~~~~~~~
+
+You can use `pre-commit`__ to easily add
+``cmake-format`` as a pre-commit hook in your git repository. Once
+you `have it installed`__, add this to the ``.pre-commit-config.yaml`` in your
+repository:
+
+.. code:: yaml
+
+    repos:
+    -   repo: https://github.com/cheshirekow/cmake_format.git
```

Comment:
As far as I can tell, it sounds like pre-commit will [download from github][1] and install with setup.py. I would prefer users install from pypi.org as the canonical distributions. Is that possible to configure?

[1]: https://pre-commit.com/#new-hooks